README.md

```diff
@@ -217,6 +218,25 @@ newFetchInstance(`https://${server}/${path}/?payload=` + payload)
 
 > *Adding use cases is a WIP - if you find this proposal useful (or just want to explore some potential use cases), help by visiting [#4](https://github.com/weizman/Realms-Initialization-Control/issues/4) and share your use case to push this proposal forward!*
 
+## Value
+
+While this feature is developers facing, the value it aspires to introduce is for the end users really, because until this proposal lands, the same origin concern prevents developers from building safe composable web applications within their own origin and instead place untrusted code within cross origin realms which affects the end user in 2 major ways:
```

Comment:
"The feature aims to enable developers to provide better user experiences when embedding untrusted code into their applications.

Currently when embedding untrusted code, developers are using cross-origin iframes or workers for that purpose, which often creates separate experiences from their own content.

While developers can restrict their own documents (by overriding various JS functions and DOM APIs), attackers can use same-origin realms to regain access to these prototypes. 

This proposal enables developers to prevent attackers from regaining access to native prototypes, and hence embed untrusted code into their documents in a safe way. This provides users with richer and more coherent embedded experiences on the web" ??

Reply:
76d5bd5c7fe6bd15113ff14085eb06c54cc9c799